tests/lost_reset/test

```diff
@@ -41,69 +45,77 @@ for ( $i = 0 ; $i < 10 ; $i++ ) {    # iteration count of 10
     ###
     # tests
     # Start floodping to generate activity
-    seek( $fh_tmp, 0, 0 );
-    system("ping -f 127.0.0.1 >/dev/null 2>&1 & echo \$! >$tmpout");
-    my $ping_pid = <$fh_tmp>;
+    seek( $fh_out, 0, 0 );
+    system("ping -f 127.0.0.1 >/dev/null 2>&1 & echo \$! >$stdout");
+    my $ping_pid = <$fh_out>;
     chomp($ping_pid);
 
     # Add rule to generate audit queue events from floodping
     $result =
       system("auditctl -a exit,always -S all -F pid=$ping_pid >/dev/null 2>&1");
-    ok( $result, 0 );    # Was the rule added successfully?
-    sleep 1;
+    sleep 2;
     kill 'TERM', $ping_pid;
     system("auditctl -d exit,always -S all -F pid=$ping_pid >/dev/null 2>&1");
 
     # Restart the daemon to collect messages in the log
     system("service auditd start >/dev/null 2>&1");
-    sleep 1;
 }
 
-# send the reset lost message (NOTE: requires bash)
-seek( $fh_tmp, 0, 0 );
-$result = system(
-    "echo \$BASHPID\ >$tmpout; exec auditctl --reset-lost >$stdout 2>$stderr");
+sleep 1;
+
+my $status_lost;
+if ( defined $ENV{ATS_DEBUG} && $ENV{ATS_DEBUG} == 1 ) {
+    seek( $fh_out, 0, 0 );
+    system(
+        "echo -n \"auditctl -s: \" >$stdout; auditctl -s|grep lost >>$stdout");
+    $status_lost = <$fh_out>;
+}
+
+# send the reset lost message
+seek( $fh_out, 0, 0 );
+$result = system("auditctl --reset-lost >$stdout 2>$stderr");
 ok( $result, 0 );    # Was the reset command successful?
-my $reset_lost_pid = <$fh_tmp>;
-chomp($reset_lost_pid);
-my $reset_err = <$fh_err>;
-$reset_err =~ /lost: ([0-9]+)/;
-my $result_lost = $1;
-ok( $result_lost > 0 );    # Was the lost value non-zero?
+<$fh_err> =~ /lost: ([0-9]+)/;
+my $reset_rc = $1;
+ok( $reset_rc > 0 );    # Was the lost value non-zero?
 
 sleep 1;
 
 # find the config change event
 seek( $fh_out, 0, 0 );
 seek( $fh_err, 0, 0 );
-$result = system("ausearch -ts recent -i -m CONFIG_CHANGE >$stdout 2>$stderr");
-ok( $result, 0 );          # Was an event found?
+$result = system(
+"LC_TIME=\"en_DK.utf8\" ausearch --start $startdate $starttime -i -m CONFIG_CHANGE >$stdout 2>$stderr"
```

Comment:
Why do we need to set LC_TIME here, especially since we aren't setting it above?

Reply:
From @rgbriggs
> So that it works consistently on any system ....

Gotcha.